src/main/kotlin/validation/TryCountValidation.kt

```diff
@@ -1,26 +1,29 @@
 package validation
 
 object TryCountValidation {
-    fun checkTryCount(tryCount: String) {
+    private const val MINIMUM_TRY_COUNT = 1
+
+    private const val TRY_COUNT_INTEGER_ERROR_MESSAGE = "[ERROR] 시도 횟수는 숫자만 입력해 주세요"
+    private const val TRY_COUNT_POSITIVE_ERROR_MESSAGE = "[ERROR] 시도 횟수는 양의 정수여야 합니다"
+    fun validate(input: String) {
```

Comment:
```suggestion
    private const val TRY_COUNT_POSITIVE_ERROR_MESSAGE = "[ERROR] 시도 횟수는 양의 정수여야 합니다"
    
    fun validate(input: String) {
```
변수 정의부의 마지막과 함수 정의부 초반부에는 개행으로 구분해주시는 것이 좋습니다. (이하 관련 내용 동일)